packages/scripts/src/generate-master-list/generator.ts

```diff
@@ -64,18 +65,36 @@ const getEndpoints = async (adapterPath: string, verbose = false) => {
   let batchableEndpoints = 'Unknown'
   try {
     const indexPath = adapterPath + '/src/endpoint/index.ts'
+    const schemaPath = adapterPath + '/src/schemas/env.json'
 
     const endpointDetails: EndpointDetails = await require(path.join(process.cwd(), indexPath))
-
+    const schema = getJsonFile(schemaPath) as Schema
```

Comment:
Let's make it so for adapters using the new version, we don't add a schema for those, it was an old initiative but we've gone in a different direction. We can use its lack of existence as a way to determine the major version